packages/components/src/components/languageSupport/lezerDebug.ts

```diff
@@ -0,0 +1,105 @@
+import { Input, NodeType, Tree, TreeCursor } from "@lezer/common";
+
+enum Color {
+  Red = 31,
+  Green = 32,
+  Yellow = 33,
+}
+
+function colorize(value: any, color: number): string {
+  return String(value);
+  // return "\u001b[" + color + "m" + String(value) + "\u001b[39m"
+}
+
+function focusedNode(cursor: TreeCursor): {
```

Comment:
What's the purpose of this if it just destruct some properties in cursor? Couldn't you just access the original cursor object?